cirq-core/cirq/protocols/json_serialization_test.py

```diff
@@ -232,7 +232,7 @@ def test_builtins():
 def test_numpy():
     x = np.ones(1)[0]
 
-    assert_json_roundtrip_works(x.astype(np.bool))
+    assert_json_roundtrip_works(x.astype(np.bool_))
```

Comment:
maybe add a comment why this is interesting? It seems like a private type.

Reply:
I read more, and I don't think we need this check, the objects we will get will al be dtype bool, so we should be same.